package_constraint_solver/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: constraint_solver
 description: A general-purpose backtracking constraint satisfaction algorithm.
-version: 1.0.0
 repository: https://github.com/dart-lang/samples/tree/main/package_constraint_solver
+publish_to: none
```

Comment:
@johnpryan – saw this is discontinued. Not planning to publish any more, right?

Reply:
Yeah I don't plan on updating the pub package anymore, but I'd prefer to keep the CI separate since it was part of this sample.

Reply:
We should at least remove the broken badge from the [README](https://github.com/dart-lang/samples/tree/main/package_constraint_solver#readme) for now